core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -104,11 +104,11 @@ var StoryEditorPage = function() {
     by.css('.protractor-test-warnings-text'));
   var storyThumbnailImageElement = element(
     by.css('.story-thumbnail .protractor-test-custom-photo'));
-  var storyThumbnailClickable = element(
+  var storyThumbnailButton = element(
     by.css('.story-thumbnail .protractor-test-photo-clickable'));
```

Comment:
Perhaps update the protractor class name as well, here and elsewhere. ("clickable" isn't really a "thing".)

Reply:
Done.